### PR TITLE
INS-3219: try to maintain logger with object/request fields

### PR DESCRIPTION
**Pull request by ruz**
improve setting logger and context for transcript

### CODE REVIEW

File: logicrunner/handle_additional_call_test.go [hunk 1]
```diff
@@ -87,7 +86,7 @@ func TestHandleAdditionalCallFromPreviousExecutor_Present(t *testing.T) {
 				require.NoError(t, err)
 			}
 
-			mc.Wait(2 * time.Minute)
+			mc.Wait(2 * time.Second)
```
Comment (bigbes) [inline logicrunner/handle_additional_call_test.go]: second?

Comment (aalekseev-ru) [inline logicrunner/handle_additional_call_test.go]: fix this pls

Comment (ruz) [reply logicrunner/handle_additional_call_test.go]: fixed

Comment (ruz) [reply logicrunner/handle_additional_call_test.go]: fixed

File: logicrunner/common/transcript.go [hunk 2]
```diff
@@ -60,3 +98,40 @@ func (t *Transcript) HasOutgoingRequest(
 	}
 	return nil
 }
+
+func contextFromServiceData(data payload.ServiceData) context.Context {
```
Comment (sexyhamster) [inline logicrunner/common/transcript.go]: see common.ContextWithServiceData and FreshContextFromContext

Comment (ruz) [reply logicrunner/common/transcript.go]: no such methods

Comment (sexyhamster) [reply logicrunner/common/transcript.go]: then it is copypasted from package logicrunner. In my PR i moved them to common.

Comment (ruz) [reply logicrunner/common/transcript.go]: which PR? what do you suggest me to do here? I made sure that nobody else uses these functions? should I wait for you PR to be merged or rebase on top of it?

Comment (sexyhamster) [reply logicrunner/common/transcript.go]: my bad, missed new NewTranscriptCloneContext

**Comment by ita-sammann** [comment]
Unit test passed. Force merging.

**Pull request merged**